assets/app/scripts/filters/resources.js

```diff
@@ -845,4 +845,56 @@ angular.module('openshiftConsole')
 
       return portDisplayValue(servicePort.port, servicePort.targetPort, servicePort.protocol);
     };
+  })
+  .filter('sentenceCase', function() {
+    // Converts a camel case string to sentence case
+    return function(str) {
+      if (!str) {
+        return str;
+      }
+
+      // Unfortunately, _.lowerCase() and _.upperFirst() aren't in our lodash version.
+      var lower = _.startCase(str).toLowerCase();
+      return lower.charAt(0).toUpperCase() + lower.slice(1);
+    };
+  })
+  .filter('podStatus', function() {
+    // Return results that match kubernetes/pkg/kubectl/resource_printer.go
+    return function(pod) {
+      if (!pod || (!pod.deletionTimestamp && !pod.status)) {
+        return '';
```

Comment:
There's a pod phase "Unknown" that has a specific meaning, so hesitant to do that.